dotnet/src/Connectors/Connectors.OpenAI/AzureSdk/ClientCore.cs

```diff
@@ -1036,7 +1036,7 @@ private static CompletionsOptions CreateCompletionsOptions(string text, OpenAIPr
             Echo = false,
             ChoicesPerPrompt = executionSettings.ResultsPerPrompt,
             GenerationSampleCount = executionSettings.ResultsPerPrompt,
-            LogProbabilityCount = null,
+            LogProbabilityCount = executionSettings.LogProbabilitiesCount,
```

Comment:
Should we use the same name for the property?

Reply:
Azure SDK uses `LogProbabilityCount` for text generation and `LogProbabilitiesPerToken` for chat completion. I decided to go with `LogProbabilitiesCount` as something in the middle.

Reply:
Renamed to `Logprobs` and `TopLogprobs`.